### PR TITLE
Translate "Asset Id" when viewing version of user notes category

### LINKED ISSUE #15166: Undefined language string in Versions
### Steps to reproduce the issue
Under the following section, edit an entry and preview a version to see text without translation (highlighted in yellow).

Users > User Note Categories:
user-notes

### PR DESCRIPTION
**Pull request by PhilETaylor**
Translate "Asset Id" when viewing version of user notes category  (Like anyone even uses this feature! :-( )

Users > User Note Categories -> Edit -> Versions -> Preview 

Pull Request for Issue Closes #15166 .

### Summary of Changes

Language string missing - now not :)

### CODE REVIEW

File: administrator/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -185,6 +185,8 @@ JFIELD_ALT_LAYOUT_LABEL="Layout"
 JFIELD_ALT_MODULE_LAYOUT_DESC="Use a layout from the supplied module or overrides in the templates."
 JFIELD_ALT_PAGE_TITLE_DESC="An optional alternative page title to set that will change the TITLE tag in the HTML output."
 JFIELD_ALT_PAGE_TITLE_LABEL="Alternative Page Title"
+JFIELD_ASSET_ID_LABEL="Asset ID"
+JFIELD_ASSET_ID_DESC="Asset ID"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.ini]: Please swap lines so they are in alphabetical order. Thanks.

**Comment by brianteeman** [comment]
i wonder why we are even displaying the asset_id at all as we dont in any other version history eg categories

**Comment by mbabker** [comment]
My first thought too.  Seems like a system detail that honestly I don't think needs tracked in history unless the ACL state is too and that ID definitely doesn't need to be shown to the user (aside from the debug reports, where are we actually showing asset IDs in the admin UI?).

**Comment by brianteeman** [comment]
My vote would be to remove it from this view as we dont show it anywhere else

File: administrator/language/en-GB/en-GB.ini (hunk 1 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.ini]: i fixed this with https://github.com/joomla/joomla-cms/pull/15840/commits/b2708738ce82883c452b4858bffb29d930962a16

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on b2708738ce82883c452b4858bffb29d930962a16<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15840">issues.joomla.org/tracker/joomla-cms/15840</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b2708738ce82883c452b4858bffb29d930962a16<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15840">issues.joomla.org/tracker/joomla-cms/15840</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Pull request merged**